scripts/generate_mintlify_openapi_docs.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+import json
+import subprocess
+import sys
+from pathlib import Path
+from typing import Any, Union
+
+from prefect.server.api.server import create_app
+
+Mint = dict[str, Any]
+Navigation = list[dict[str, Any]]
+SinglePage = str
+PageGroup = dict[str, Any]
+
+
+def docs_path() -> Path:
+    return Path(__file__).parent.parent / "docs"
+
+
+def current_version() -> str:
+    # TODO: derive this from the docs and the current version of the prefect package
+    return "3.0rc"
```

Comment:
Gotcha.

Reply:
Yeah this is something I want to dig more into.  I can make it work on `<major>.<minor>` for now (adding the `rc` tag if appropriate)